bika/lims/content/sampletype.py

```diff
@@ -198,6 +254,158 @@ def ContainerTypesVocabulary(self):
         from bika.lims.content.containertype import ContainerTypes
         return ContainerTypes(self, allow_blank=True)
 
+    def getDefaultSmallSticker(self):
+        """
+        Returns the small sticker ID defined as default.
+
+        :return: A string as an sticker ID
+        """
+        values = self.getField('AdmittedStickerTemplates').get(self)
+        if not values:
+            return ''
+        value = values[0].get('small_default')
+        return value
+
+    def getDefaultLargeSticker(self):
+        """
+        Returns the large sticker ID defined as default.
+
+        :return: A string as an sticker ID
+        """
+        values = self.getField('AdmittedStickerTemplates').get(self)
+        if not values:
+            return ''
+        value = values[0].get('large_default')
+        return value
+
+    def getAdmittedStickers(self):
+        """
+        Returns the admitted sticker IDs defined.
+
+        :return: An array of sticker IDs
+        """
+        values = self.getField('AdmittedStickerTemplates').get(self)
+        if not values:
+            return []
+        admitted = values[0].get('admitted')
+        return admitted
```

Comment:
Almost the same exact logic as `getDefaultSmallSticker` and `getDefaultLargeSticker`. Better to create the following functions:

```python
def getDefaultSticker(self, size=STICKER_SMALL):
    return self.get_sticker_subfield(size)

def getAdmittedStickers(self):
    return self._get_sticker_subfield('admitted', default=list())

def _get_sticker_subfield(subfield, default=None):
    if subfield is None:
        return default
    values = self.getAdmittedStickerTemplates()
    if not values:
        return default
    return values[0].get('admitted', default)    
```